.gitignore

```
@@ -185,7 +185,8 @@ cython_debug/
 .LSOverride
 
 # Icon must end with two \r
-Icon

+Icon
```

keep \r\n

---

Editorconfig automatically removes whitespace at the end of lines, so it is difficult to keep this entry corrrect. We did a little research (read StackOverflow) and using a wildcard `?` instead is probably a reasonably safe way to do this.

```suggestion
Icon?
```